app/views/trainees/show.html.erb

```diff
@@ -1,59 +1,86 @@
-<%= form_with url: trn_submissions_path, method: :post, local: true do |f| %>
-  <%= hidden_field_tag :trainee_id, @trainee.id %>
-  <%= f.govuk_submit "Submit for TRN" %>
-<%- end -%>
 
-<h2 class="govuk-heading-l">Personal details</h2>
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: 'All records',
+    href: trainees_path
+  ) %>
+<% end %>
 
-<p class="govuk-body">
-  <%= govuk_link_to "Update", personal_details_trainee_path(@trainee) %>
-</p>
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-xl">Add a trainee</span>
+  Overview
+</h1>
 
-<div class="personal-details">
-  <%= render_component(SummaryCard::View.new(title: "Personal details",
-                                             rows: TraineePersonalDetails.new(@trainee).call))  %>
-</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_component TaskList::View.new(classes: "record-setup") do |component|
+      component.slot(
+        :row,
+        task_name: 'Record setup',
+        path: "",
+        status: 'completed'
+      )
```

Comment:
I'm still wrapping my head around view components and the govuk bits, but isn't the whole point of using `wrap_slot :row` to be able to just this:

```suggestion
      row task_name: 'Record setup',
          path: "",
          status: 'completed'
```